src/components/common.js

```diff
@@ -546,7 +546,7 @@ export const ClipboardButton = ({ text, onClick, ...props }) => {
   const [copied, setCopied] = useState(false)
   return h(Link, {
     ...props,
-    tooltip: 'Copy to clipboard',
+    tooltip: copied ? 'Copied to clipboard' : 'Copy to clipboard',
```

Comment:
#2525 is necessary in order to resolve the original issue identified by aXe where this button has no `aria-label`.